phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/MainWindowView.java

```diff
@@ -66,15 +67,21 @@ public MainWindowView(String text, ThemeManager themeManager) {
         this.content.getStyleClass().add("mainWindowScene");
 
         this.populateFailurePanel();
-        this.populateWaitPanel();
+        this.waitPanel = createWaitPanel();
 
         this.content.setCenter(waitPanel);
 
         this.setContent(content);
     }
 
-    private void populateWaitPanel() {
-        this.waitPanel = new WaitPanel();
+    private HBox createWaitPanel() {
```

Comment:
With this implementation, the wait panel can only be used in `MainWindowView`. Not sure if that's important or not.

Reply:
The wait panel was only used in the `MainWindowView`.
Otherwise it would be worth a thought to add a separate control and skin class for it.

Reply:
We can always do that later if we need to.